packages/perseus-editor/src/widgets/numeric-input-editor.tsx

```diff
@@ -446,175 +610,188 @@ class NumericInputEditor extends React.Component<Props, State> {
+                                            value: firstNumericalParse(
+                                                newValue,
+                                                this.context.strings,
+                                            ),
+                                        });
+                                    }}
+                                />
+                                <span className="max-error-plusmn">
+                                    &plusmn;
+                                </span>
+                                <NumberInput
+                                    className="max-error-input-value"
+                                    placeholder={0}
+                                    value={answers[i]["maxError"]}
+                                    format={_.last(answer.answerForms || [])}
+                                    onChange={this.updateAnswer(i, "maxError")}
+                                />
+                            </div>
+                            <fieldset className="perseus-widget-row">
+                                <legend className="inline-options">
+                                    Status:
+                                </legend>
+                                <RadioOption
+                                    answerIndex={i}
+                                    answerProperty="status"
+                                    value="correct"
+                                    onClick={() => {
+                                        this.onEvaluationChange(i, "correct");
+                                    }}
+                                >
+                                    Correct
+                                </RadioOption>
+                                <RadioOption
+                                    answerIndex={i}
+                                    answerProperty="status"
+                                    value="wrong"
+                                    onClick={() => {
+                                        this.onEvaluationChange(i, "wrong");
+                                    }}
+                                >
+                                    Wrong
+                                </RadioOption>
+                                <RadioOption
+                                    answerIndex={i}
+                                    answerProperty="status"
+                                    value="ungraded"
+                                    onClick={() => {
+                                        this.onEvaluationChange(i, "ungraded");
+                                    }}
+                                >
+                                    Ungraded
+                                </RadioOption>
+                            </fieldset>
+                            {unsimplifiedAnswers(i)}
+                            <div className="perseus-widget-row">
+                                Message shown to user in article:
```

Comment:
I wish we had a way to identify which editor view we're in, as it'd be lovely to hide this option when adding Numeric Inputs in exercises.